lld/MachO/Symbols.h

```diff
@@ -41,6 +41,11 @@ class Symbol {
     AliasKind,
   };
 
+  // Enum that describes the kind of ICF folding that a symbol is involved in.
+  // We need to keep track of this to correctly display symbol sizes in the map
+  // file.
+  enum ICFFoldKind { None, Folded_Body, Folded_Thunk };
```

Comment:
The prefix `Folded_` seems redundant. Can you clarify the comments for each case something like:
```
	// Enum that describes the type of Identical Code Folding (ICF) applied to a symbol.
	// This information is crucial for accurately representing symbol sizes in the map file.
enum ICFFoldKind {
  None,    // No folding is applied.
  Body,    // The entire body (function or data) is folded.
  Thunk    // The function body is folded into a thunk.
};
```